content/docs/user-guide/scripting/lua/_index.md

```diff
@@ -6,18 +6,24 @@ toc: true
 weight: 100
 ---
 
-**Topics**
-+ [Adding Lua Scripts to Component Entities](/docs/user-guide/editor/add-lua-script)
-+ [Properties Table](properties/)
-+ [Using EBuses in Lua](ebus/)
-+ [Lua Editor](lua-editor/)
-+ [Debugging Lua Scripts](debugging-scripts/)
-+ [The Lua Environment (Advanced)](environment/)
-
-You can use Lua in O3DE to facilitate quick iteration of your game project. Lua is a powerful, fast, lightweight, embeddable scripting language. When you construct new gameplay and game systems, you can run your changes immediately, without compiling your source code.
+You can use Lua in **Open 3D Engine (O3DE)** to facilitate quick iteration of your game project. Lua is a powerful, fast, lightweight, embeddable scripting language. When you construct new gameplay and game systems, you can run your changes immediately, without compiling your source code.
 
 O3DE uses Lua version {{< versions/lua >}}.
 
+## Building Lua Editor and GridHub
+
+**Lua Editor** (Lua IDE) and GridHub, which is O3DE's central connection hub for debugging, must be built for every project that requires them. Include `LuaIDE` and `GridHub` as build targets when you build your project: `--target <project>.GameLauncher Editor LuaIDE GridHub`.  
```

Comment:
Need to qualify this build instruction, because LuaIDE and GridHub come pre-built if you build O3DE using the install layout, or install O3DE from the installer.
```suggestion
If you built O3DE from source, you must explicitly include **Lua Editor** (Lua IDE) and GridHub as build targets when you build your project. GridHub is O3DE's central connection hub for debugging, and is required to debug Lua scripts from the Lua IDE. To include these targets, add `LuaIDE` and `GridHub` to your build target list: `--target <project>.GameLauncher Editor LuaIDE GridHub`. 
```